src/components/NewLocationPage/ChartTabs/ChartTab.style.tsx

```diff
@@ -0,0 +1,48 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+export const TabsContainer = styled.div`
+  display: flex;
+`;
+
+export const Tab = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-width: 4rem;
+  max-width: 10.5rem;
```

Comment:
removing `min-width` and `max-width`, and adding `width: fit-content` should do the trick